openapi/components/schemas/key-value-stores/CreateKeyValueStoreResponse.yaml

```diff
@@ -1,9 +1,9 @@
-title: Getlistofkey-valuestoresresponse
+title: CreateKeyValueStoreResponse
```

Comment:
No idea why, but git is mixing up these three files. 🤦 

`GetListOfKeysResponse` -> `UpdateStoreResponse` -> `CreateKeyValueStoreResponse`

Reply:
:sweat_smile: